docs/connecting-to-a-lit-network/mainnets.md

```diff
@@ -0,0 +1,48 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Mainnets
+
+Lit mainnets are designed for late-stage development and production deployment. If you are implementing storing assets with real world value, it should be done on the main networks and not the test networks. While main networks may be deprecated in the future, assets will be transferable onto new networks.
+
+
+Here is an overview of the Lit mainnets:
+
+| Name       | Lit Blockchain                                                   | Description                                                                                                                                                                         | Minimum Lit SDK Version | Lit SDK Network Identifier | Requires Payment |
+|------------|------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|----------------------------|------------------|
+| Datil-prod | [Chronicle Yellowstone](./lit-blockchains/chronicle-yellowstone) | Centralized mainnet designed for production deployment. Guaranteed real world asset transferability onto new mainnets. Payment is enforced.                                                                        | `^6.4.0`                | `datil-prod`               | ✅                |
```

Comment:
This should say 'decentralized mainnet' (not centralized)

Reply:
Also just to avoid confusion, i think we can edit this
> Guaranteed real world asset transferability onto new mainnets

To just say 'keys are persistent and won't be deleted'